datadog_checks_dev/tests/tooling/test_license_headers.py

```diff
@@ -199,6 +200,48 @@ def test_validate_license_headers_returns_error_on_existing_file_with_changed_he
     assert len(errors) == 1
     assert errors[0].message == "existing file has changed license"
     assert errors[0].path == "setup.py"
+    assert errors[0].fixed == f"{original_license}\n\nimport sys\n"
+
+
+def test_validate_license_headers_accepts_any_header_when_previous_version_with_no_license_exists(tmp_path):
```

Comment:
Feels like a java style name 😏 (just joking, the purpose of the test is clear and that's nice)

Reply:
Utilizing the max line length to the limit 😆. On a serious note, it's true that I could consider wrapping the tests into a class (or more) to reduce some of the repetition in the names... 🤔.

Reply:
Yeah that would make sense but it's not a blocker